openid-federation-1_0.xml

```diff
@@ -9825,7 +9825,7 @@ Host: op.umu.se
 	-40
 	<list style="symbols">
 	  <t>
-	    TBD
+	    Fixed #90: Corrected request object audience example.
```

Comment:
since it was for a non normative example, I would keep it out from the changelog